forest/daemon/src/daemon.rs

```diff
@@ -399,7 +399,7 @@ async fn maybe_fetch_snapshot(should_fetch_snapshot: bool, config: Config) -> Co
         let path = match snapshot_fetch(
             &snapshot_path,
             &config,
-            &SnapshotServer::Filecoin,
+            &Some(SnapshotServer::Forest),
```

Comment:
This won't work for downloading a mainnet snapshot. How about using `None`?

Reply:
fixed